src/permission.js

```diff
@@ -99,13 +99,16 @@
       function handleAuthorizedState() {
 
         TransitionEvents.broadcastStateChangePermissionAccepted();
-        $location.replace();
 
         // Overwrite notify option to broadcast it later
         TransitionProperties.options = angular.extend({}, TransitionProperties.options, {notify: false});
 
         $state
-          .go(TransitionProperties.toState.name, TransitionProperties.toParams, TransitionProperties.options)
+          .go(
+            TransitionProperties.toState.name,
+            TransitionProperties.toParams,
+            angular.extend({}, TransitionProperties.options, {location: 'replace'})
```

Comment:
I can't because this overrides `TransitionProperties.options` and I want to
leave that as it is
On 20 Apr 2016 21:59, "Błażej Krysiak" <notifications@github.com> wrote:

> In src/permission.js
> <https://github.com/Narzerus/angular-permission/pull/219#discussion_r60486467>
> :
>
> >
> >          // Overwrite notify option to broadcast it later
> >          TransitionProperties.options = angular.extend({}, TransitionProperties.options, {notify: false});
> >
> >          $state
> > -          .go(TransitionProperties.toState.name, TransitionProperties.toParams, TransitionProperties.options)
> > +          .go(
> > +            TransitionProperties.toState.name,
> > +            TransitionProperties.toParams,
> > +            angular.extend({}, TransitionProperties.options, {location: 'replace'})
>
> You can move this part to line 104 to object with notify properties and
> add location there to clean up the code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Narzerus/angular-permission/pull/219/files/504dc0bcb4ec885083288ca23f4b03c20994e5e5#r60486467>
>